client/src/PublicRequest.ts

```diff
@@ -313,13 +361,18 @@ export type SignSwapRequestStandard = SignSwapRequestCommon & {
 
 export type SignSwapRequestSlider = SignSwapRequestCommon & {
     layout: 'slider',
+    direction: 'left-to-right' | 'right-to-left',
     nimiqAddresses: Array<{
         address: string,
         balance: number, // Luna
     }>,
     bitcoinAccount: {
         balance: number, // Sats
     },
+    polygonAddresses: Array<{
+        address: string,
+        balance: number, // cents
```

Comment:
Is it really in cents? If yes, then it would be the first time we're not using the smallest unit, right?
And at the same time it's a bit counter intuitive to use cents for polygon (which use Matic and Wei).

Just pointing it out, not asking for changes.

Reply:
Oh, no, you are right, it's not in Cents, it's in USDC smallest unit. What should we call that?

Reply:
It apparently doesn't have a name, but Polygon and ETH smallest unit is Wei, so maybe we can call is usdc-wei or smth the like.